x-pack/plugins/security_solution/public/timelines/store/timeline/reducer.ts

```diff
@@ -530,6 +531,18 @@ export const timelineReducer = reducerWithInitialState(initialTimelineState)
       },
     },
   }))
+  .case(setDataProviderVisibility, (state, { id, isDataProviderVisible }) => {
```

Comment:
I haven't looked at the way we test these reducers, but shouldn't we add some unit tests when introducing a new case?

Reply:
Currently we do not test `reducers`, I think it is because they are very atomic piece of functionality where we are simple using external library to modify the state.

However, there are some complex actions which make use of some `helpers` when reducing the state. There was a test file for those called `reducer.test.tsx`. I have renamed that to `helpers.test.tsx` in  af2d7372da4e3a90820781cbc60322d48e224002